pkg/components/exporter/exporter.go

```diff
@@ -99,17 +107,10 @@ func NewDaemonSet(k *v1alpha1.Kepler) *appsv1.DaemonSet {
 					Labels:    podSelector,
 				},
 				Spec: corev1.PodSpec{
-					NodeSelector:       linuxNodeSelector,
+					NodeSelector:       linuxNodeSelector.Merge(nodeSelector),
 					ServiceAccountName: ServiceAccountName,
 					DNSPolicy:          corev1.DNSPolicy(corev1.DNSClusterFirstWithHostNet),
-					Tolerations: []corev1.Toleration{
-						{
-							Effect: corev1.TaintEffectNoSchedule, Key: "node-role.kubernetes.io/master",
-						},
-						{
-							Effect: corev1.TaintEffectNoSchedule, Key: "node-role.kubernetes.io/infra",
-						},
-					},
+					Tolerations:        append(defaultTolerations, tolerations...),
```

Comment:
I am not really sure of  ... I think we should apply the  toleration **only** if the user has not specified a toleration.
What if I want my deployment to run on normal nodes , i.e. do not run on master (for whatever reason).

Reply:
users can add another taint to exclude master nodes in that case